core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -441,7 +443,7 @@ private List<FlowWithTriggers> computeSchedulable(List<FlowWithSource> flows, Li
 
         return flows
             .stream()
-            .map(flow -> pluginDefaultService.injectDefaults(flow, log))
+            .map(flow -> pluginDefaultService.injectAllDefaults(flow, log))
```

Comment:
Don't we already have the defaults as they are injected inside the flow listener consumer?